daemons/fenced/fenced_commands.c

```diff
@@ -349,7 +349,7 @@ create_async_command(xmlNode *msg)
         return NULL;
     }
 
-    op = get_xpath_object("//@" PCMK__XA_ST_DEVICE_ACTION, msg, LOG_ERR);
+    op = get_xpath_object("//*[@" PCMK__XA_ST_DEVICE_ACTION "]", msg, LOG_ERR);
```

Comment:
Look carefully through this commit please, and anything else that modifies actual XPath expressions. We're not going to get a compile time error for bad XPath syntax, or for good syntax that no longer matches what we want to.